master/getting-started/bare-metal/bare-metal.md

```diff
@@ -479,28 +480,80 @@ outside the cluster:
    then be accepted or dropped according to workload policy (if it is going to
    a local workload) or to normal host endpoint policy (if not).
 
+## Policy on forwarded traffic
+If `applyOnForward` is `false`, host endpoint policy applies to traffic to/from
```

Comment:
I think it will be clearer if you add a "the" before "host endpoint policy", both here and below. It appears that this setting is contained within the policy itself. If I am right, then adding the "the" will make this more clear. Otherwise, the setting could be somewhere else and affect all host endpoint policies.